docs/querying/api.md

```diff
@@ -433,13 +433,35 @@ URL query parameters:
   series from which to read the label values. Optional.
 - `limit=<number>`: Maximum number of returned series. Optional. 0 means disabled.
 
-
 The `data` section of the JSON response is a list of string label values.
 
-This example queries for all label values for the `job` label:
+This example queries for all label values for the `http_status_code` label:
+
+```json
+$ curl http://localhost:9090/api/v1/label/http_status_code/values
+{
+   "status" : "success",
+   "data" : [
+      "node",
+      "prometheus"
```

Comment:
I small nit that can be addressed in a follow up: `node` and `prometheus` are not `http_status_codes`.  The example is of course still valid but might give people pause. There is another instance of this further down.

Reply:
ah good point